### ISSUE TITLE
MissingMethodException: Method not found: 'NearColliderLogic.GetBuildingsInAreaNonAlloc'.

### ISSUE THREAD

**Issue by Chalos-Atiesh** [fezhub/DSP-Mods#38]
https://imgur.com/QEbRStA

Get this any time I'm shift+click copying an assembler with the newest update.

**Comment by fezhub** [comment]
Please update to CopyInserters 1.7, which fixes this issue

**Comment by Chalos-Atiesh** [comment]
Hmm, I did check for updates. For some reason r2modman didn't detect it. Let me try again.

**Comment by Chalos-Atiesh** [comment]
That did it. Thanks and great work! Closing.
